runtime/ops/fs.rs

```diff
@@ -1826,16 +1933,19 @@ fn op_make_temp_file_sync(
 }
 
 #[op]
-async fn op_make_temp_file_async(
+async fn op_make_temp_file_async<P>(
```

Comment:
FWIW, I find putting the constraint with the type parameter more easy to read `<P: FsPermissions + 'static>`

Reply:
I don't think that will work, clippy is complaining:
```
error[E0658]: associated type bounds are unstable
   --> runtime/ops/fs.rs:198:17
    |
198 | fn op_open_sync<P: FsPermissions + 'static>(
    |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #52662 <https://github.com/rust-lang/rust/issues/52662> for more information

```

Reply:
Oh, ok too bad for now.